site/_posts/2020-06-03-user-registration-and-sign-in-with-laravel.md

```diff
@@ -64,6 +69,10 @@ FUSIONAUTH_BASE_URL=http://localhost:9011
 Finally, you can take advantage of [Laravel's dependency injection system](https://laravel.com/docs/7.x/providers) by registering the `FusionAuthClient` in the `boot()` method of your `app/Providers/AppServiceProvider.php` file:
```

Comment:
Did we switch versions of Laravel? Which version did you test against. Might want to note that here.

Reply:
Laravel Framework 7.30.4 
- I will add a note to the GH repo too 👍🏻